src/library/scala/util/Random.scala

```diff
@@ -65,11 +75,23 @@ class Random(val self: java.util.Random) extends AnyRef with Serializable {
    */
   def nextInt(n: Int): Int = self.nextInt(n)
 
+  /** Returns a pseudorandom, uniformly distributed int value between min
+   *  (inclusive) and the specified value max (exclusive), drawn from this
+   *  random number generator's sequence.
+   */
+  def between(min: Int, max: Int): Int = min + self.nextInt(max - min)
+
   /** Returns the next pseudorandom, uniformly distributed long value
    *  from this random number generator's sequence.
    */
   def nextLong(): Long = self.nextLong()
 
+   /** Returns a pseudorandom, uniformly distributed long value between min
+   *  (inclusive) and the specified value max (exclusive), drawn from this
+   *  random number generator's sequence.
+   */
+  def between(min: Long, max: Long): Long = min + self.nextLong() % (max - min)
```

Comment:
Also I did a test to see the uniformity of `min + Math.abs(r.nextInt()) % (max - min) and the values between min and max were uniform. Coincidence? Or am I missing something